docs/source/implementations/labeling_trend_scanning.rst

```diff
@@ -0,0 +1,41 @@
+.. _implementations-labeling_trend_scanning:
+
+==========================================
+Labeling: Trend Scanning
+==========================================
+
+Trend scanning is a classification labeling technique introduced by Marcos Lopez de Prado in the following lecture slides: "Advances in Financial Machine Learning: Lecture 3/10"
+'https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3257419'
```

Comment:
This url can be added as a link. Please see the PortOpt modules for a good example. Its better to have a link of a word than a long url.